package.json

```diff
@@ -43,7 +43,8 @@
     "android": "(cd ./native && npm run android)",
     "check-owners": "npm whoami && grep -E $(npm whoami) NPM_OWNERS && node scripts/npm/check-owners.js",
     "publish": "npm run check-owners && npm run build && git pull --rebase && npm run test && lerna publish",
-    "danger": "danger"
+    "danger": "danger",
+    "update-bpk-dependencies": "node scripts/npm/update-bpk-dependencies.js"
```

Comment:
this doesn't actually update anymore does it? I suggest `"check-bpk-dependencies"` && `check-bpk-dependencies.js` respectively.

Reply:
Cool!